lib/suppressions.h

```diff
@@ -115,6 +116,14 @@ class CPPCHECKLIB Suppressions {
      */
     std::string parseXmlFile(const char *filename);
 
+    /**
+     * Parse multi inline suppression in comment
+     * @param comment the full comment text
+     * @param errorMessage output parameter for error message (wrong suppression attribute)
+     * @return empty vector if something wrong.
+     */
+    std::vector<Suppression> parseMultiSuppressComment(std::string comment, std::string *errorMessage);
```

Comment:
it seems to me this method can be `static`

Reply:
yes, but it seems not harmonious to parseXmlFile and parseFile ^_^. but I added it.